docs/reference/repo_properties.rst

```diff
@@ -0,0 +1,162 @@
+
+Package Repository Properties
+*****************************
+
+.. _ppa-properties:
+PPA properties
+==============
+
+The following properties are supported for PPA-type repositories:
+
+- type
+   - Type: enum[string]
+   - Description: Specifies type of package-repository, must currently be
+     ``apt``
+   - Examples: ``type: apt``
+- ppa
+   - Type: string
+   - Description: PPA shortcut string
+   - Format: <ppa-owner>/<ppa-name>
+   - Examples:
+      - ``ppa: snappy-devs/snapcraft-daily``
+      - ``ppa: mozillateam/firefox-next``
+
+
+.. _deb-properties:
+Deb properties
+==============
+
+The following properties are supported for Deb-type repositories:
+
+- architectures
+   - Type: list[string]
+   - Description: Architectures to enable, or restrict to, for this repository
+   - Default: If unspecified, architectures is assumed to match the host’s
+     architecture
+   - Examples:
+      - ``architectures: [i386]``
+      - ``architectures: [i386, amd64]``
+- components
+   - Type: list[string]
+   - Description: Apt repository components to enable: e.g.
+     ``main``, ``multiverse``, ``unstable``
+   - Examples:
+       - ``components: [main]``
+       - ``components: [main, multiverse, universe, restricted]``
+- formats
+   - Type: list[string]
+   - Description: List of deb types to enable
+   - Default: If unspecified, format is assumed to be ``deb``, i.e. ``[deb]``
+   - Examples:
+       - ``formats: [deb]``
+       - ``formats: [deb, deb-src]``
+- key-id
+   - Type: string
+   - Description: 40 character GPG key identifier ("long-form thumbprint" or
+     "fingerprint")
+     If not using a key-server, Snapcraft will look for the corresponding key
```

Comment:
Is this supposed to still say Snapcraft?

Reply:
Nope! fixed thanks (used "Craft Application" instead)